### ISSUE TITLE
Warning: Each child in a list should have a unique "key" prop.

### ISSUE THREAD

**Issue by hazem3500** [aholachek/storybook-mobile#11]
When including the addon it shows the error `react.development.js:312 Warning: Each child in a list should have a unique "key" prop.` on startup, is there a way to fix it?
I am using storybook v6.0.21

**Comment by guyzmo** [comment]
I also got a crash:

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': 'label for="react-select-2-input"' is not a valid selector.
    at Array.map (<anonymous>)
    at attachLabels
    at getInputTypeWarnings
    at Hints
    at renderWithHooks
    at updateFunctionComponent 
    at beginWork
    at HTMLUnknownElement.callCallback 
    at Object.invokeGuardedCallbackDev 
```

followed by:

```
Check the top-level render call using <Styled(div)>. See https://fb.me/react-warning-keys for more information.
    in Fragment
    in Unknown
    in Unknown
    in Unknown
    in Unknown (created by ManagerConsumer)
    in ManagerConsumer (created by Panel)
    in Panel (created by Layout)
    in div (created by Context.Consumer)
    in Styled(div) (created by Panel)
    in Panel (created by Layout)
    in div (created by Context.Consumer)
    in Styled(div) (created by Main)
    in div (created by Context.Consumer)
    in Styled(div) (created by Main)
    in Main (created by Layout)
    in Layout (created by Context.Consumer)
    in WithTheme(Layout)
    in Unknown
    in div (created by Context.Consumer)
    in Styled(div)
    in Unknown (created by SizeMeRenderer(Component))
    in SizeMeReferenceWrapper (created by SizeMeRenderer(Component))
    in SizeMeRenderer(Component) (created by SizeMe(Component))
    in SizeMe(Component)
    in ThemeProvider
    in Unknown (created by ManagerConsumer)
    in ManagerConsumer (created by Manager)
    in EffectOnMount (created by Manager)
    in Manager (created by Context.Consumer)
    in Location (created by QueryLocation)
    in QueryLocation (created by Root)
    in LocationProvider (created by Root)
    in HelmetProvider (created by Root)
    in Root
```

This only happens when I try to open stories that use `react-select`.

I had to uninstall storybook-mobile to keep on working 😕

**Comment by aholachek** [comment]
Hi @guyzmo, I've fixed the react-select bug in v0.1.25, thank you for catching it! But unfortunately I'm not really sure what's causing the key warning, since there are no obvious suspects without keys in the storybook-mobile addon's components.

**Comment by domyen** [comment]
@aholachek is this the error message you mentioned needing a hand with?

**Comment by aholachek** [comment]
Hi @domyen, yep! I believe it actually might be related to [this similar issue from the now-deprecated notes icon](https://github.com/storybookjs/storybook/issues/7005). In any case, I can't seem to figure out what I might be doing wrong in my code, if anything, since all list items should have keys.

**Comment by winkerVSbecks** [comment]
@aholachek I think I found the issue here: https://github.com/aholachek/storybook-mobile/blob/f5fe27dd736a377d9f8897b9b3ae53ea929b5068/src/register.js#L85

I believe you need to add the `key` to the `React.Fragment` wrapper itself. 

```js
addons.register(ADDON_ID, () => {
  const render = ({ active, key }) => {
    return (
      <React.Fragment key={key}>
        <ViewportManager active={active} />
        <AddonPanel active={active} key={key}>
          <StateWrapper active={active}>
            <MyPanel key={key} active={active} />
          </StateWrapper>
        </AddonPanel>
      </React.Fragment>
    )
  }
```

**Comment by aholachek** [comment]
@winkerVSbecks that fixes it! thank you! I'll release a patch tomorrow.

**Comment by aholachek** [comment]
this should be fixed in v0.1.26
